docs/UserGuide.md

```diff
@@ -136,9 +138,9 @@ Adds a student to the address book.
 
 <box type="tip" header="##### Tips">
 
-
-* New clashing schedule will be informed so that you can modify using the [`edit` command](#editing-a-student-edit).
-* <b>ADDRESS</b> can be used to store place of tuition. E.g. You can store tutee's address if the tuition happens at their place or you can store `My Place` if the tuition is at your place.   
+* <b>ADDRESS</b> can be used to store place of tuition. E.g. You can store tutee's address if the tuition happens at their place or you can store `My Place` if the tuition is at your place.
+* UGTeach will inform you of clashing schedules so that you can modify using the [`edit` command](#editing-a-student-edit).
```

Comment:
> * UGTeach will inform you of clashing schedules so that you can modify using the [`edit` command](#editing-a-student-edit).

Maybe split into two sentence;
UG will inform you of the clashing schedules.
You can modify them using the edit command.

Reply:
Thanks for the suggestion! Ive made the changes